src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -150,7 +139,8 @@ The .NET Foundation licenses this file to you under the MIT license.
       <PrivateSdkAssemblies Include="$(IlcPath)\sdk\*.dll" />
     </ItemGroup>
     <ItemGroup>
-      <FrameworkAssemblies Include="$(IlcPath)\framework\*.dll" />
+      <!-- Exclude System.IO.Compression.Native.dll for now https://github.com/dotnet/corert/issues/5496 -->
```

Comment:
```suggestion
      <!-- Exclude System.IO.Compression.Native.dll -->
```
I do not think that the link to that issue is relevant. That issue is fixed, just not marked as such because of the corert repo is archived.